src/lib/actor.js

```diff
@@ -53,15 +53,15 @@ const getApifyStorageClient = (options = {}, forceCloud = false) => {
  * @return {string}
  */
 const getDefaultStorageId = (storeType) => {
-    const isRunningOnApify = !process.env[ENV_VARS.LOCAL_STORAGE_DIR];
-    const envVarName = ENV_VARS[`DEFAULT_${storeType}_ID`];
+    const isRunningOnApify = !process.env[APIFY_ENV_VARS.LOCAL_STORAGE_DIR]; // TODO: wtf is this??? we have "IS_AT_HOME"...
+    const envVarName = ACTOR_ENV_VARS[`DEFAULT_${storeType}_ID`];
     const storeId = process.env[envVarName];
-    if (isRunningOnApify && !storeId) {
+    if (isRunningOnApify && !storeId) { // TODO: this condition means it wil complain when running in the cloud? wtf?
```

Comment:
I think this condition is just us being too cautious, and it would never happen. The right env vars will always be set by the worker. So we could just simplify this and remove the platform vs local logic, and keep just lines 57, 58 and 64.

Reply:
I performed the simplification